modules/vactory_contact/vactory_contact.install

```diff
@@ -0,0 +1,66 @@
+<?php
+
+use Drupal\menu_link_content\Entity\MenuLinkContent;
+use Drupal\node\Entity\Node;
+use Drupal\paragraphs\Entity\Paragraph;
+
+/**
+ * @file
+ * Install file.
+ */
+
+function vactory_contact_install() {
+
+  // Create page that contains contact form.
+  $contact_widget_id = 'vactory_decoupled_webform:webform';
```

Comment:
`vactory_decoupled_webform:webform` may change, this wil break, try catch or define your own df